cmd/ebrelayer/relayer/cosmos.go

```diff
@@ -377,10 +401,47 @@ func (sub CosmosSub) handleBurnLockMsg(cosmosMsg types.CosmosMsg, claimType type
 	sub.SugaredLogger.Infow("get the prophecy claim.",
 		"CosmosSender", prophecyClaim.CosmosSender,
 		"CosmosSenderSequence", prophecyClaim.CosmosSenderSequence)
+		
+	client, auth, target, err := tryInitRelayConfig(sub, claimType)
+	if err != nil {
+		sub.SugaredLogger.Errorw("failed in init relay config.",
+			errorMessageKey, err.Error())
+		return
+	}
 
-	err := txs.RelayProphecyClaimToEthereum(sub.EthProvider, sub.RegistryContractAddress,
-		claimType, prophecyClaim, sub.PrivateKey, sub.SugaredLogger)
+	// Initialize CosmosBridge instance
+	cosmosBridgeInstance, err := cosmosbridge.NewCosmosBridge(target, client)
 	if err != nil {
-		log.Println(err)
+		sub.SugaredLogger.Errorw("failed to get cosmosBridge instance.",
+			errorMessageKey, err.Error())
```

Comment:
Don't we have to return here?  cosmosBridgeInstance isn't valid, so we can't pass it to RelayProphecyClaimToEthereum

Reply:
Return added.